hubspot3/tickets.py

```diff
@@ -15,6 +20,13 @@ class TicketsClient(BaseClient):
     :see: https://developers.hubspot.com/docs/methods/tickets/tickets-overview
     """
 
+    class Recency:
+        """recency type enum"""
+
+        CREATED = "CREATED"
```

Comment:
why caps?  Other files are using lower case

Reply:
this is defined by the API (which is way less consistent than desired)